Parsers/DSTIM/GetClassificationList.txt

```diff
@@ -0,0 +1,12 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as GetClassificationList.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. GetClassificationList | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let GetClassificationList = (_startDate:datetime, _endDate: datetime, _subscriptions:dynamic = dynamic("*"), _resourceGroups:dynamic = dynamic(["*"]), _userAccounts:dynamic = dynamic(["*"])) 
+{ 
+    DSTIMCorrelatedLogs(_dateStart, _dateEnd, _subscriptions, _resourceGroups, _userAccounts)
```

Comment:
It seems parameter names like _startDate,_endDate should be same in function.Thanks

Reply:
@v-sumdh fixed.